client/src/components/App.js

```diff
@@ -17,13 +17,20 @@ import { authorizeJWT } from '../actions/auth';
 const App = () => {
   const { logIn } = useSelector((state) => state.auth);
   const dispatch = useDispatch();
-
+  const { toasts } = useToasterStore();
   useEffect(() => {
     if (logIn) {
       dispatch(authorizeJWT());
     }
   }, [dispatch, logIn]);
 
+  useEffect(() => {
+    toasts
+      .filter((t) => t.visible) // Only consider visible toasts
+      .filter((_, i) => i >= 1) // Is toast index over limit 2
+      .forEach((t) => toast.dismiss(t.id)); // Dismiss – Use toast.remove(t.id) removal without animation
```

Comment:
These comments don't make much sense to me. Consider adding a comment on top of the `useEffect` that links to this implementation: https://github.com/timolins/react-hot-toast/issues/31#issuecomment-803359550